src/modules/Dropdown/Dropdown.js

```diff
@@ -755,6 +756,38 @@ export default class Dropdown extends Component {
     return _.findIndex(options, ['value', value])
   }
 
+  getDropdownAriaOptions = (ElementType) => {
+    const { loading, disabled, search, multiple } = this.props
+    const { open } = this.state
+    const ariaOptions = {
+      role: 'listbox',
+      'aria-busy': loading,
+      'aria-disabled': disabled,
+      'aria-expanded': !!open,
+    }
+    if (search) {
+      ariaOptions.role = 'combobox'
+    } else if (ElementType === Menu.Item) {
+      ariaOptions.role = 'menuitem'
+    } else {
+      ariaOptions['aria-multiselectable'] = multiple
```

Comment:
Also would like to have this up top with the object, just so all the aria props are in one location.  Note that a multi-select dropdown may or may not be searchable.  I suspect the aria prop for multiple should be added in both cases.

It could be that this should only be applied when the Dropdown is a `select` Dropdown?  The difference being that the normal Dropdown is the "menu only style" and the `select` prop creates behavior like a `<select />` element.

LMK